packages/code-du-travail-frontend/src/outils/IndemniteLicenciement/components/IndemniteConventionnelle.js

```diff
@@ -1,6 +1,7 @@
 import React from "react";
 import MathJax from "react-mathjax-preview";
-import { Alert, Button } from "@cdt/ui";
+import styled from "styled-components";
```

Comment:
I propose to limit the modifications here to the minimum (add AdditionnalContent) since this screen is currenctly being refactor

Reply:
Unluckily, this one is mendatory. I have to display several warnings in the result page. I'll handle the conflict :/

Reply:
I know I propose to limit the changes to the addAdditionnalWarning

Reply:
read too quickly, sure !